src/components/Note/note.jsx

```diff
@@ -31,26 +54,78 @@ const triggerEdit=(event,item)=>{
   showAddNote(true, "EDIT")
 }
 
+async function restoreToNote(event,item){
+  event.preventDefault();
+  try{
+    const response= await axios.post(`/api/archives/restore/${item._id}`,
+      {},
```

Comment:
Why is this empty object passed here?

Reply:
it was required in the API call

Reply:
If the body is empty then, It's not necessary to pass an empty obj, you can omit it.